scripts/new_project.kts

```diff
@@ -103,6 +107,32 @@ object NewProject {
             }
     }
 
+    private fun renameApplicationClass() {
+        showMessage("=> 🔎 Renaming application class...")
+        File(projectPath)
+            ?.walk()
+            .find { it.name == "$TEMPLATE_APPLICATION_CLASS_NAME.kt" }
+            ?.let { templateApplicationFile ->
+                val newApplicationPath = templateApplicationFile.absolutePath.replaceAfterLast(
+                    delimiter = fileSeparator,
+                    replacement = "$applicationClassName.kt"
+                )
+                val newApplicationFile = File(newApplicationPath)
+                templateApplicationFile.renameTo(newApplicationFile)
+            }
+
+        File(projectPath)
+            ?.walk()
+            .filter { it.name.endsWith(".kt") || it.name.endsWith(".xml") }
```

Comment:
@Tuubz Which file of .kt that we have to rename here 🙏

Reply:
The application class file, ex: `CoroutineTemplateApplication.kt` 🚀

Reply:
What about in line `121` 🤔 Is it duplicate 🙏

Reply:
Oh, I see 👍 

The first part is to rename the file name, but the second part is for inside the file:

![Screen Shot 2565-05-20 at 17 39 01](https://user-images.githubusercontent.com/18277915/169511063-31867fa8-fd20-430b-b646-49fbe2398cd8.png)

Reply:
Oh, I got it now  🚀

Reply:
@Tuubz I think we can do like this :

![Screen Shot 2565-05-23 at 19 15 41](https://user-images.githubusercontent.com/28002315/169817227-d1921cf3-4b9a-46d4-945b-c4c8c3c5deb5.png)

What do you think? 🙏

Reply:
Sounds good, I'll re-factor 🚀 

I agree, it would be better for easier understanding